pod/perl5380delta.pod

```diff
@@ -0,0 +1,1658 @@
+
+=item L<C<newSV_true()>|perlapi/newSV_true>
+
+=item L<C<newSV_false()>|perlapi/newSV_false>
+
+=item L<C<sv_set_true(SV *sv)>|perlapi/sv_set_true>
+
+=item L<C<sv_set_false(SV *sv)>|perlapi/sv_set_false>
+
+=item L<C<sv_set_bool(SV *sv, const bool bool_val)>|perlapi/sv_set_bool>
+
+=back
+
+The macros are:
+
+=over 4
+
+=item L<C<SvIandPOK(sv)>|perlapi/SvIandPOK>
+
+=item L<C<SvIandPOK_off(sv)>|perlapi/SvIandPOK_off>
+
+=item L<C<SvIandPOK_on>|perlapi/SvIandPOK_on>
+
+=back
+
+=item *
+
+Perl is no longer manipulating the C<environ> array directly. The variable
+C<PL_use_safe_putenv> has been removed and C<PERL_USE_SAFE_PUTENV> is always
+defined. This means XS modules can now call C<setenv> and C<putenv> without
+causing segfaults. [L<perl #19399|https://github.com/Perl/perl5/issues/19399>]
+
+=item *
+
+Internal C API functions are now hidden with C<__attribute__((hidden))> on the
+platforms that support it. This means they are no longer callable from XS
+modules on those platforms.
+
+It should be noted that those functions have always been hidden on Windows. This
+change merely brings that to the other platforms.
+[L<perl #19655|https://github.com/Perl/perl5/pull/19655>]
+
+=item *
+
+New formatting symbols were added for printing values declared as U32 or
+I32:
+
+=over
+
+=item I32df -- Like %d
+
+=item U32of -- Like %o
+
+=item U32uf -- Like %u
+
+=item U32xf -- Like %x
+
+=item U32Xf -- Like %X
+
+=back
+
+These are used in the same way already existing similar symbols, such as
+C<IVdf>, are used.  See L<perlapi/I/O Formats>.
+
+=item *
+
+new 'HvHasAUX' macro
+
+=item *
+
+regexec.c: Add some branch predictions reorder conds
+
+=item *
+
+locale: Change macro name to be C conformant
+
+=item *
+
+Rename the C<PADNAMEt_*> constants to C<PADNAMEf_*>
+
+=item *
+
+Changes all the API macros that retrieve a PV into a call to an
+inline function so as to evaluate the parameter just once.
+
+=item *
+
+regexec.c: multiple code refactor to make the code more readable
+
+=item *
+
+perl.h: Change macro name to be C conformant
+(remove leading _ from NOT_IN_NUMERIC macros)
+
+=item *
+
+regcomp.h: add new C<BITMAP_BIT> macro in addition to the existing C<BITMAP_BYTE>
+and C<BITMAP_TEST> ones.
+
+=item *
+
+Create new regnode type ANYOFH.
+populate_ANYOF_from_invlist was renamed to populate_bitmap_from_invlist
+
+=item *
+
+regex: Refactor bitmap vs non-bitmap of qr/[]/
+
+=item *
+
+regcomp.c: add new functions to convert from an inversion list to a bitmap (and vice versa)
+C<populate_bitmap_from_invlist> and C<populate_invlist_from_bitmap>.
+
+=item *
+
+Add C<newAVav()> to create an AV from an existing AV.
+Add C<newAVhv()> to create an AV using keys and values from an existing HV.
+
+=item *
+
+Fix definition of C<Perl_atof>.
+
+=item *
+
+Fix undefined behavior with overflow related OPTIMIZE_INFTY and delta
+in C<regcomp.c>.
+
+=item *
+
+Fix regnode pointer alignment issue in <regcomp.h>.
```

Comment:
Presumably C<> around the header name, not just <>, though this and immediately previous mention of regcomp.c should probably be F<> instead of C<>.

Reply:
Done, thanks.